From ed0c417db235c391484c35eb7101d8e4329a2da0 Mon Sep 17 00:00:00 2001
From: "Zeeshan Ali (Khattak)" <zeeshanak@gnome.org>
Date: Sun, 16 Mar 2014 15:20:19 +0000
Subject: [PATCH 2/3] media-manager: Ignore CDROM devices

Apparently Qemu is unable to handle CDROM devices so lets not present
them to user.
---
 src/media-manager.vala | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/media-manager.vala b/src/media-manager.vala
index 902f1b3..a27d958 100644
--- a/src/media-manager.vala
+++ b/src/media-manager.vala
@@ -77,8 +77,9 @@ public async InstallerMedia create_installer_media_for_path (string       path,
         enumerator.add_match_property ("DEVTYPE", "disk");
 
         foreach (var device in enumerator.execute ()) {
-            if (device.get_property ("ID_FS_BOOT_SYSTEM_ID") == null &&
-                !device.get_property_as_boolean ("OSINFO_BOOTABLE"))
+            if ((device.get_property ("ID_FS_BOOT_SYSTEM_ID") == null &&
+                 !device.get_property_as_boolean ("OSINFO_BOOTABLE")) ||
+                 device.get_property_as_boolean ("ID_CDROM")) // Qemu has issues with CD-ROM devices
                 continue;
 
             var path = device.get_device_file ();
-- 
1.8.5.3

